### DISCUSSION TITLE
[gatsby-starter-portfolio-jodie] How to duplicate Projects section

### DISCUSSION THREAD

**Discussion by IonizedCaffiene** [LekoArts/gatsby-themes#827] [Questions] [answered]
How would I go about creating a duplicate of the Projects section for a different set of content? (e.g. have one "Projects" set for Photos and one for Design, keeping them as separate sets)

**Comment by trevorcmit** [comment]
@IonizedCaffiene Have you been able to fix this? I am trying to do a similar thing, and have been able to add to the navigation side panel (simple to do in the gatsby-config.js) but am running into issues with the placement of a .tsx or .js that represents the page and can pull from the components in other sections.

**Comment by LekoArts** [comment, marked as answer]
I'd do it something along those lines:

1. Create a `src/templates/design.tsx`
2. Copy the contents of https://github.com/LekoArts/gatsby-themes/blob/main/themes/gatsby-theme-jodie/src/components/projects.tsx and adjust the imports to be coming from `@lekoarts/gatsby-theme-jodie`
3. Set up a new `gatsby-source-filesystem` pointing to your `design` files
4. Run `createPages` API inside `gatsby-node.ts` to query `allFile` and filter for `sourceInstanceName` on the name that you gave to the `gatsby-source-filesystem` in step 3. This way you only get the relevant MDX files
5. In the `createPage` action you use the template from step 1
6. Adjust the navbar to include the new page
